app/controllers/libraries_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LibrariesController < ApplicationController
+
+  def index
+    render :index
```

Comment:
Rails will automatically render the view with the name of the method, so you can omit this line